pages/en/lb4/Application.md

```diff
@@ -32,13 +32,17 @@ tasks as a part of your setup:
 import {Application} from '@loopback/core';
 import {RestComponent, RestServer} from '@loopback/rest';
 import {SamoflangeController, DoohickeyController} from './controllers';
-import {WidgetApi} from './apidef/';
 
 export class WidgetApplication extends Application {
```

Comment:
IMO, you should extend `RestApplication` here. The only reason why you would not want to, would be if your application was exposing multiple REST servers, which is not the case AFAICT.

This will also allow you to get rid of `app.component(RestComponent)` below.

Reply:
The only problem I have with that is the page the docs is about: `Application`. IMO an example shown in this page should be vanilla `Application` instead of `RestApplication`. What do you think?

Reply:
Fair enough, let's keep using `Application` then 👍 

Could you perhaps extend [Tips for application setup](https://github.com/strongloop/loopback.io/blob/d903c1b26965386f5aca8092206ca93b98eebb0b/pages/en/lb4/Application.md#tips-for-application-setup) to mention `RestApplication` + when and why to use it? It's ok to leave that out of this pull request though.